src/helpers/sceneless-project.ts

```diff
@@ -352,16 +352,79 @@ export const commands = (_project: ScenelessProject) => {
   const background = root.children.find((x) => x.props.id === 'bg')
   const content = root.children.find((x) => x.props.id === 'content')
   const foreground = root.children.find((x) => x.props.id === 'foreground')
-  const bannerContainer = foreground.children.find(
+
+  const coreProject = getProject(_project.id)
+
+  let bannerContainer = foreground?.children?.find(
     (x) => x.props.id === 'fg-banners',
   )
-  const foregroundImageContainer = foreground.children.find(
+
+  let foregroundImageContainer = foreground?.children?.find(
     (x) => x.props.id === 'fg-image',
   )
-  const foregroundVideoContainer = foreground.children.find(
+
+  let foregroundVideoContainer = foreground?.children?.find(
     (x) => x.props.id === 'fg-video',
   )
 
+  const ensureForegroundContainers = async () => {
```

Comment:
not sure if there is a better place to do this; will consult hudson when he comes back.